fix get all changed files gitutil (#2990)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Add the `--staged-only` flag to **pre-commit**.

> * Fixed an issue where **pre-commit** command collected all files in some environments.

Reviewer: ```suggestion
* Fixed an issue where **pre-commit** ran on unchanged files.
```
